plugins/destination/gremlin/client/spec.go

```diff
@@ -33,6 +33,9 @@ type Spec struct {
 	// AWS region to use for AWS IAM authentication. Required when `auth_mode` is `aws`.
 	AWSRegion string `json:"aws_region"`
 
+	// AWS Neptune host header to use with AWS IAM authentication. Required when `auth_mode` is `aws`.
+	AWSNeptuneHost string `json:"aws_neptune_host" jsonschema:"pattern=^[\\w\\.-]+$"`
```

Comment:
This is completely optional (if you're running a sync inside the same VPC as your Neptune, you wouldn't need it) but it still needs to be just hostname-valid-chars. No slash, no `scheme://`.